packages/paste-website/src/pages/components/checkbox/index.mdx

```diff
@@ -54,17 +54,21 @@ export const getStaticProps = async () => {
 
 ## Guidelines
 
-### Checkbox Group
+### About Checkbox Group
 
-<Paragraph>{meta.description}</Paragraph>
+Use Checkbox Groups when you have multiple binary choices to make but all choices belong to a single group or field. Example: Choosing capabilities when searching for a phone number to provision from Twilio. By placing them in a group, users should be able to clearly understand which options make up the collection and how those options are related to each other.
 
-Checkbox groups are used when you have multiple binary choices to make but all choices belong to a single group or field. For example, choosing capabilities when searching for a Phone Number to provision from Twilio. By placing them in a group, users should be able to clearly understand which options make up the collection and also how those options are related to each other.
+For guidance on using a Checkbox in a form, refer to our [Form pattern](/patterns/form) guidelines.
+
+### About Checkbox
+
+Use a checkbox to present a user with a binary (e.g. on/off) choice that is part of a form. A checkbox should not be used to apply a user's choice instantly. In that case, such as enabling a system feature, use the [Switch component](/components/switch).
 
 #### Accessibility
 
-- Checkbox groups **must** have a group legend that describes the collection.
-  - The label _should_ be visible.
-  - The label may be visually hidden or provided via `aria-label` on the group component if the entire form is just a single checkbox group with no other form elements. The grouping should be visually implicit, but a description still needs to be provided for assistive technology.
+- Checkbox Group must have a group legend that describes the collection. The label should be visible. However if the entire form is just a single checkbox group with no other form elements, it may be visually hidden or provided via `aria-label` on the group component. The grouping should be visually implicit, but a description still needs to be provided for assistive technology.
+- Checkbox must have a visible label. 
+- When displaying additional content based on checking a checkbox, be sure that the new content appears after the checkbox so that it is naturally discoverable by assistive technology users.
 
```

Comment:
chore: Remove the sections below ("Checkbox" and "Accessibility") since they repeat what's above it.